### ISSUE TITLE
OOM error in standalone wiremock 2.27.1

### ISSUE THREAD

**Issue by saucam** [tomakehurst/wiremock#1375]
This is a regression from 2.26.3 and can be observed in version 2.27.1 in standalone wiremock and standalone jre8 wiremock as well
## Bug reports

We're a lot more likely to look at and fix bugs that are clearly described and reproduceable. Please including the following details when reporting a bug:

- [ ] Which version of WireMock you're using
2.27.1 (both jre8 and non-jre standalone)

- [ ] How you're starting and configuring WireMock, including configuration or CLI the command line
Using JRE11
```
java -server -Xms2G -Xmx2G -XX:+PrintGCDetails -XX:G1HeapRegionSize=4M -jar /path/standalone.jar --disable-gzip --no-request-journal
```

- [ ] A failing test case that demonstrates the problem

Submitted a few requests that was sending form-data with 2 jpeg files in the request

- [ ] Profiler data if the issue is performance related   

GC logs show old gen is exhausted

```
[63.880s][info   ][gc,phases      ] GC(125) Phase 1: Mark live objects 11.054ms
[63.880s][info   ][gc,phases,start] GC(125) Phase 2: Prepare for compaction
[63.882s][info   ][gc,phases      ] GC(125) Phase 2: Prepare for compaction 1.867ms
[63.882s][info   ][gc,phases,start] GC(125) Phase 3: Adjust pointers
[63.893s][info   ][gc,phases      ] GC(125) Phase 3: Adjust pointers 11.016ms
[63.893s][info   ][gc,phases,start] GC(125) Phase 4: Compact heap
[63.902s][info   ][gc,phases      ] GC(125) Phase 4: Compact heap 8.853ms
[63.905s][info   ][gc,heap        ] GC(125) Eden regions: 0->0(25)
[63.905s][info   ][gc,heap        ] GC(125) Survivor regions: 0->0(4)
[63.905s][info   ][gc,heap        ] GC(125) Old regions: 512->511
[63.905s][info   ][gc,heap        ] GC(125) Humongous regions: 0->0
[63.905s][info   ][gc,metaspace   ] GC(125) Metaspace: 18600K->18600K(1067008K)
[63.905s][info   ][gc             ] GC(125) Pause Full (G1 Evacuation Pause) 1751M->1749M(2048M) 35.760ms
[63.905s][info   ][gc,cpu         ] GC(125) User=0.06s Sys=0.00s Real=0.04s
[63.913s][info   ][gc,start       ] GC(126) Pause Young (Concurrent Start) (G1 Evacuation Pause)
[63.913s][info   ][gc,task        ] GC(126) Using 4 workers of 4 for evacuation
[63.919s][info   ][gc             ] GC(126) To-space exhausted
[63.919s][info   ][gc,phases      ] GC(126)   Pre Evacuate Collection Set: 0.0ms
[63.919s][info   ][gc,phases      ] GC(126)   Evacuate Collection Set: 5.8ms
[63.919s][info   ][gc,phases      ] GC(126)   Post Evacuate Collection Set: 0.5ms
[63.919s][info   ][gc,phases      ] GC(126)   Other: 0.2ms
[63.919s][info   ][gc,heap        ] GC(126) Eden regions: 1->0(25)
[63.919s][info   ][gc,heap        ] GC(126) Survivor regions: 0->0(4)
[63.919s][info   ][gc,heap        ] GC(126) Old regions: 511->512
[63.919s][info   ][gc,heap        ] GC(126) Humongous regions: 0->0
[63.919s][info   ][gc,metaspace   ] GC(126) Metaspace: 18600K->18600K(1067008K)
```

jmap shows a lot of byte arrays
```
$ jmap -histo:live 11003
 num     #instances         #bytes  class name (module)
-------------------------------------------------------
   1:         39678     1829127840  [B (java.base@11.0.7)
   2:          2038        3377016  [I (java.base@11.0.7)

```

This issue can be reproduced with both G1 GC and CMS

**Comment by tomakehurst** [comment]
This is the same issue as described here I think: https://groups.google.com/g/wiremock-user/c/kwWDtutVBIo/m/VWysfmwbAQAJ

I'm working on a fix at the moment and I'll be putting a patch release out soon to resolve it.

**Comment by saucam** [comment]
@tomakehurst thanks!

**Comment by tomakehurst** [comment]
This fix is released in 2.27.2

**Comment by saucam** [comment]
awesome thanks!
